src/bin/cargo/commands/test.rs

```diff
@@ -34,7 +34,7 @@ pub fn cli() -> Command {
             "Test all tests",
             "Test only the specified bench target",
             "Test all benches",
-            "Test all targets",
+            "Test all targets except doctests",
```

Comment:
> >  I would probably word this like (does not include doctests) or something like that, but I am splitting hairs and the exact wording isn't important to me, just a suggestion if you like it better.

I like the suggestion for it being paranthetical

I think I also prefer wording besides "except" makes it sound like we are actively excluding it
- `does not include` is ok
- `precludes` might also work

Reply:
I can switch to `does not include` with a parenthetical.

Though, one thing that phrasing doesn't address or even hint at is that `cargo test` without this flag _does_ run doctests. From that perspective, adding this flag does have the effect of actively excluding doctests, even though I understand the difference in intent there.